docs/tutorial/part-one/index.md

```diff
@@ -403,6 +405,28 @@ Once this finishes running, you should see in your terminal something like:
 Open the web address listed on the bottom line (`lowly-pain.surge.sh` in this
 case) and you'll see your newly published site! Great work!
 
+## Alternative: Deploying to Gatsby Cloud
+
+[Gatsby Cloud](https://gatsbyjs.com) is a platform built specifically for Gatsby sites, with features like real-time previews, fast builds, and integrations with dozens of other tools.
```

Comment:
add: "It's the best place to build and deploy sites built with Gatsby, and you can use Gatsby Cloud free for personal and community projects."